torch/quantization/fx/graph_module.py

```diff
@@ -47,8 +44,34 @@ def __deepcopy__(self, memo):
         fake_mod.__dict__ = copy.deepcopy(self.__dict__)
         return ObservedStandaloneGraphModule(fake_mod, self.graph)
 
-def mark_observed_standalone_module(module: GraphModule) -> GraphModule:
-    return ObservedStandaloneGraphModule(module, module.graph)
-
 def is_observed_standalone_module(module: Any) -> bool:
     return isinstance(module, ObservedStandaloneGraphModule)
+
+
+class QuantizedGraphModule(GraphModule):
+    def __init__(self, root: Union[torch.nn.Module, Dict[str, Any]], graph: Graph):
+        super().__init__(root, graph)
+
+    def _save_to_state_dict(self, destination, prefix, keep_vars):
+        super()._save_to_state_dict(destination, prefix, keep_vars)
+        for attr_name in dir(self):
+            if "_packed_weight" in attr_name and \
+               isinstance(getattr(self, attr_name), torch._C.ScriptObject):  # type: ignore
+                packed_weight = getattr(self, attr_name)
+                destination[prefix + attr_name] = packed_weight
+
+    def _load_from_state_dict(self, state_dict, prefix, local_metadata, strict,
+                              missing_keys, unexpected_keys, error_msgs):
+        attrs_to_pop = []
+        for attr_name in state_dict:
+            if "_packed_weight" in attr_name and isinstance(state_dict[attr_name], torch._C.ScriptObject):  # type: ignore
+                setattr(self, attr_name, state_dict[attr_name])
```

Comment:
Should these be registered as Parameters in any way?

Reply:
these are not trainable. packed params are the packed quantized weight + bias after quantization. for example, it's generated by:
```
self._packed_weight = torch.ops.quantized.linear_prepack(quantized_weight, bias)
```

ideally they should be registered as buffers, but currently buffer only support Tensor and Module I think.

Reply:
ok!